docs/fides/mkdocs.yml

```diff
@@ -99,5 +110,8 @@ extra_css:
   - css/taxonomy.css
 
 plugins:
+  # The "Last Update" footer only shows in production
+  - git-revision-date:
+      enabled_if_env: CI
```

Comment:
The variable name `CI` makes sense where it's defined, [in `.github/workflows/publish_docs.yaml`](https://github.com/ethyca/fides/pull/265/files#diff-4de995e1609d8ad989b4542c46a7789c934599bab17bab700944864a8acaee27R22), since that executes in our CI environment, but it makes this config confusing. Since the `publish_docs` workflow only runs on pushes to the `main` branch, I think it's equally intuitive to call the variable `PROD` or something. Thoughts?

Reply:
good call, this could be more self-documenting